package.json

```
@@ -342,12 +342,12 @@
     "lint-eslint": "yarn build:cljs && yarn lint-eslint-pure",
     "lint-eslint-pure": "eslint --ext .js,.jsx,.ts,.tsx --rulesdir frontend/lint/eslint-rules --max-warnings 0 --report-unused-disable-directives enterprise/frontend/src frontend/src frontend/test e2e/test",
     "lint-prettier": "yarn && yarn lint-prettier-pure",
-    "lint-prettier-pure": "prettier '{enterprise/,}frontend/**/*.{js,jsx,ts,tsx,css}' --check",
+    "lint-prettier-pure": "prettier '{frontend,enterprise,e2e}/**/*.{js,jsx,ts,tsx,css}' --check",
```

I'm not an expert in doing file patterns, but what I've tested is

```suggestion
    "lint-prettier-pure": "prettier '{{enterprise/,}frontend,e2e}/**/*.{js,jsx,ts,tsx,css}' --check",
```

This is supposed to keep the previous directory search
- `enterprise/frontend/*`
- `frontend/*`
- and add a new one `e2e/*`

The difference to the change in this PR is only `enterprise/*` vs `enterprise/frontend/*`

I think the impact would be minimal, but I'd like to keep just add `e2e` directory to the search if possible and not changing the other directories settings.

---

Notice that `lint-staged` scripts are working with the entire `enterprise` directory, not `enterprise/frontend`, so now it will be more in sync (still not perfectly though).

> I think the impact would be minimal, but I'd like to keep just add e2e directory to the search if possible and not changing the other directories settings.

Makes sense but only if we change `lint-staged` scripts too.

It's best to merge this PR ASAP to prevent other people from experiencing the problem.
I'll submit a follow-up PR that will align `scripts` and `lint-staged` scripts to use the same paths.

Sounds good?

---

Sounds good. Glad to have this PR go into master ASAP